services/config_data_sources/json_project_data_source.rb

```diff
@@ -40,6 +40,27 @@ def self.map_enumerable_type(enumerable_property_name: nil, current_json_object:
         job_trigger
       end
     end
+
+    def self.json_to_attribute_name_proc_map
```

Comment:
Could you add a comment above each of these methods with a description of what it's used for?

Reply:
These are `JSONConvertible` methods, the module includes documentation, should I add more context in those implementations or leave as is?